style.css

```
@@ -0,0 +1,79 @@
+body{
+    background-image: url("");
+    width: 100vw;
+    height: 100vh;
+}
+
+main{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+}
+
+form{
+    margin-top: 10%;
+    justify-self: center;
+    height: 8%;
+    display: flex;
+}
+
+input{
+    width: 80%;
+    height: 30%;
+    font-size: 30px;
+}
+
+button{
+    height:36%;
+    width: 100px;
+}
+
+table{
+    width: 70%;
+    height: 40%;
+}
+
+th{
+    border: 1px solid black;
+    height: 5%;
+    padding: 10px;
+    background-color: aquamarine;
+}
+
+tr :nth-child(0){
+    border-right: 1px solid;
+}
+
+tr :nth-child(1){
+    border-right: 1px solid;
+}
+
+tr :nth-child(2){
+    border-right: 1px solid;
+}
+
+table {
+    border: 1px solid black;
+}
+
+td{
+    text-align: center;
+    transition: all 0.6s;
```

Se quiseres "diminuir" basta fazeres:
```suggestion
    transition: 0.6s;
```
Mas é apenas preferências